apis/v1beta1/gateway_types.go

```diff
@@ -549,9 +552,20 @@ const (
 	// true.
 	GatewayReasonProgrammed GatewayConditionReason = "Programmed"
 
-	// This reason is used with the "Programmed" condition when the Listener is
+	// This reason is used with the "Programmed" and "Accepted" condition when the Gateway is
```

Comment:
```suggestion
	// This reason is used with the "Programmed" and "Accepted" conditions when the Gateway is
```

Reply:
Fixed in https://github.com/kubernetes-sigs/gateway-api/pull/1934